### PR TITLE
Feat/classroom model post endpoint

### PR DESCRIPTION
**Pull request by AgentEnder**
Sorry for the delay on this one, had some stuff come up yesterday that was not quite expected.

### CODE REVIEW

File: OtripleS.Web.Api/Controllers/ClassroomController.cs [hunk 1]
```diff
@@ -0,0 +1,128 @@
+using System;
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Controllers/ClassroomController.cs:1]: Add copyrights notice

File: OtripleS.Web.Api/Services/Classrooms/IClassroomService.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@ namespace OtripleS.Web.Api.Services.Classrooms
 	public interface IClassroomService
 	{
 		ValueTask<Classroom> CreateClassroomAsync(Classroom classroom);
+        ValueTask<Classroom> GetClassroomById(Guid classroomId);
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Services/Classrooms/IClassroomService.cs:16]: fix your indentations

Comment (AgentEnder) [reply OtripleS.Web.Api/Services/Classrooms/IClassroomService.cs:16]: I ran code cleanup to fix these just now, there were several other files affected by the cleanup (ordering of imports in most of them).

File: OtripleS.Web.Api/Controllers/ClassroomController.cs (hunk 1 above)
Comment (AgentEnder) [reply OtripleS.Web.Api/Controllers/ClassroomController.cs]: Done

**Comment by AgentEnder** [comment]
@hassanhabib Merge issues fixed, this should be good now

File: OtripleS.Web.Api/Controllers/ClassroomsController.cs [hunk 3]
```diff
@@ -1,47 +1,47 @@
-﻿// ---------------------------------------------------------------
-// Copyright (c) Coalition of the Good-Hearted Engineers
+//�---------------------------------------------------------------
+//�Copyright�(c)�Coalition of the Good-Hearted Engineers
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Controllers/ClassroomsController.cs:2]: Something is up here with the encoding of the copyrights here.

**Pull request merged**